webapp/client/src/pages/SubmitAcknowledgePage.test.js

```diff
@@ -33,3 +26,46 @@ describe("SubmitAcknowledgePage", () => {
     );
   });
 });
+
+describe("SubmitAcknowledgePage translations", () => {
+  let c;
+
+  const submitAcknowledge =
+    i18n.getDataByLanguage("de").translation.submitAcknowledge;
+
+  beforeEach(() => {
+    c = render(
+      <I18nextProvider i18n={i18n}>
+        <SubmitAcknowledgePage {...MOCK_PROPS} />
+      </I18nextProvider>
+    );
```

Comment:
Why not just render it and use screen below as in the other tests?

Reply:
If we need to have a constant here, I would personally prefer it to have a more descriptive name

Reply:
You are right, just rendering the component with the i18 wrapper and using screen works just fine!